### ISSUE TITLE
Intermittent failure in cuML RF test with batch size 1

### ISSUE THREAD

**Issue by wphicks** [triton-inference-server/fil_backend#77]
In a manually-triggered CI run last night, we got the following failure in `test-custom`. The primary build tests passed without issue, so I'm inclined to believe this is just a flaky failure, but the reported result from Triton is obviously nonsense.

```
Starting tests of model cuml...
Performance statistics for cuml:
Traceback (most recent call last):
  File "/triton_fil/qa/L0_e2e/test_model.py", line 586, in <module>
    run_test(
  File "/triton_fil/qa/L0_e2e/test_model.py", line 461, in run_test
    np.testing.assert_almost_equal(
  File "/root/miniconda3/envs/triton_test/lib/python3.8/site-packages/numpy/testing/_private/utils.py", line 581, in assert_almost_equal
    return assert_array_almost_equal(actual, desired, decimal, err_msg)
  File "/root/miniconda3/envs/triton_test/lib/python3.8/site-packages/numpy/testing/_private/utils.py", line 1044, in assert_array_almost_equal
    assert_array_compare(compare, x, y, err_msg=err_msg, verbose=verbose,
  File "/root/miniconda3/envs/triton_test/lib/python3.8/site-packages/numpy/testing/_private/utils.py", line 842, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Arrays are not almost equal to 7 decimals
Mismatched elements: 1 / 1 (100%)
Max absolute difference: 148.23135
Max relative difference: 1.
 x: array([0.], dtype=float32)
 y: array([148.23135], dtype=float32)
ERROR conda.cli.main_run:execute(33): Subprocess for 'conda run ['bash', '/triton_fil/qa/run_tests.sh']' command failed.  (See above for error)
```

**Comment by wphicks** [comment]
This _may_ be resolved by #85. I will leave this open for now to see if it comes up again in CI. I cannot guarantee that #85 is the solution because we have only observed this failure once on a V100.

**Comment by wphicks** [comment]
Given that we have not observed this failure again since #85 was merged, I'm going to go ahead and close this for now. We can re-open  the issue if we see it again.
